content/blog/2024/09/kaoto-release-2.2.0/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Kaoto v2.2 release"
+date: 2024-09-20
+draft: false
+authors: [lordrip]
+categories: ["Releases", "Tooling", "Kaoto"]
+preview: "Kaoto 2.2 has been released"
+---
+
+<div style="text-align: center">
+  <img src="cover.png" alt="Kaoto" height="400"/>
+</div>
+
+<br/><strong>
```

Comment:
I'd avoid to have all that text in strong. Better highlight a few keywords instead.

Reply:
Removed the `<strong>` tag